Makefile

```diff
@@ -28,20 +28,61 @@ GOLANGCI_LINT_VER := v1.44.2
 GOLANGCI_LINT_BIN := golangci-lint
 GOLANGCI_LINT := $(GOBIN_DIR)/$(GOLANGCI_LINT_BIN)-$(GOLANGCI_LINT_VER)
 
+KUBE_CLIENT_GEN_VER := v0.24.0
+KUBE_CLIENT_GEN_BIN := client-gen
+KUBE_CLIENT_GEN := $(GOBIN_DIR)/$(KUBE_CLIENT_GEN_BIN)-$(KUBE_CLIENT_GEN_VER)
+
+$(KUBE_CLIENT_GEN):
+	GOBIN=$(GOBIN_DIR) $(GO_INSTALL) k8s.io/code-generator/cmd/client-gen $(KUBE_CLIENT_GEN_BIN) $(KUBE_CLIENT_GEN_VER)
+
 $(CONTROLLER_GEN):
 	GOBIN=$(GOBIN_DIR) $(GO_INSTALL) sigs.k8s.io/controller-tools/cmd/controller-gen $(CONTROLLER_GEN_BIN) $(CONTROLLER_GEN_VER)
 
+.PHONY: build
+build: ## Build the project
+	mkdir -p bin
+	go build -o bin
+
+.PHONY: install
+install:
+	go install
+
 .PHONY: codegen
-codegen: $(CONTROLLER_GEN)
+codegen: $(CONTROLLER_GEN) $(KUBE_CLIENT_GEN) build
+	# Generate deepcopy functions
 	${CONTROLLER_GEN} object paths=./testdata/pkg/apis/...
 
+	# Generate standard clientset
+	$(KUBE_CLIENT_GEN) \
+		--clientset-name versioned \
+		--go-header-file hack/boilerplate/boilerplate.generatego.txt \
+		--input-base github.com/kcp-dev/code-generator/testdata/pkg/apis \
+		--input example/v1 \
+		--output-base . \
+		--output-package github.com/kcp-dev/code-generator/testdata/pkg/generated/clientset \
+		--trim-path-prefix github.com/kcp-dev/code-generator
+
+	# Generate cluster clientset
+		bin/code-generator \
```

Comment:
nit: this is at a different indentation level

Reply:
Fixed